install/kubernetes/helm/istio/values.yaml

```diff
@@ -217,7 +217,7 @@ sidecarInjectorWebhook:
 # galley configuration
 #
 galley:
-  enabled: true
+  enabled: false
```

Comment:
I missed this change due to the PR title. Could you please cc @ozevren and myself in the future for these types of changes.

Reply:
Why do we have this change? 

+1 w/ @ayj. This should have been properly code reviewed before merging it in.

Reply:
This PR was reverted in https://github.com/istio/istio/pull/6677.

FWIW, this change did expose a bug in the post-install jobs when galley was disabled. Working on a fix for now that now.